release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -450,6 +450,43 @@ Updated boot images has been promoted to GA for Amazon Web Services (AWS) cluste
 [id="ocp-release-notes-monitoring_{context}"]
 === Monitoring
 
+The in-cluster monitoring stack for this release includes the following new and modified features. 
+
+[id="ocp-4-18-monitoring-updates-to-monitoring-stack-components-and-dependencies"]
+==== Updates to monitoring stack components and dependencies
+
+This release includes the following version updates for in-cluster monitoring stack components and dependencies:
+
+* Metrics Server to 0.7.2
+* Prometheus 2.55.1
+* Prometheus Operator to 0.78.1
+* Thanos to 0.36.1
+
+[id="ocp-4-18-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red{nbsp}Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* With this update, you can create user workload alerting and recording rules that query multiple projects at the same time.
```

Comment:
:+1:

Reply:
Sorry to unresolve this thread: Let's treat this as its own feature. We'll generate alert changes still and add them here